junit-platform-launcher/src/main/java/org/junit/platform/launcher/MethodFilter.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2015-2024 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.launcher;
+
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+
+import java.lang.reflect.Method;
+import java.util.List;
+
+import org.apiguardian.api.API;
+
+/**
+ * {@link PostDiscoveryFilter} that is applied to the fully qualified
+ * {@link Method} name without parameters.
+ *
+ * @since 1.12
+ * @see #includeMethodNamePatterns(String...)
+ * @see #excludeMethodNamePatterns(String...)
+ */
+@API(status = EXPERIMENTAL, since = "1.12")
+public interface MethodFilter extends PostDiscoveryFilter {
```

Comment:
I think this should be named `MethodNameFilter` for consistency with `ClassNameFilter`.

Reply:
I don't feel as strongly about this one anymore given it won't be next to `ClassNameFilter`. WDYT?

Reply:
I agree with you, especially there is a `TagFilter` in this package.